### PR TITLE
Auto set focus on the text area on the load

### LINKED ISSUE #5: Set the cursor on the textarea on the load
Hello Brett,

I think you have completed a really good job in implementing the Note Saver!
One thing that I noticed while testing it is that the textarea container does not automatically set the cursor on it. I believe it would be great to let the user start typing from the moment he opens your app.

Let me try to fix this for you.

### PR DESCRIPTION
**Pull request by birtony**
Fixes #5: I created a constant variable to refer to the `<textarea>` element in order to reduce code duplication. Setting cursor on the element from the load time through calling `focus()` method.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,4 +51,4 @@ <h1>Note Editor!</h1>
         }
         </script>
     </body>
-</html>
+</html>
```
Comment (UltimaBGD) [inline index.html]: There should usually be a newline at the end of files, as Github is pointing out right here.

**Review by UltimaBGD — CHANGES_REQUESTED** [review]

**Comment by UltimaBGD** [comment]
As well, it does set the focus to the text box, it does so at the beginning of the line rather than the end, is this intentional?

File: index.html (hunk 1 above)
Comment (birtony) [reply index.html]: Updated!

**Review by UltimaBGD — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (UltimaBGD) [reply index.html]: Thank you very much!

**Comment by UltimaBGD** [comment]
For now, I will close this PR and the end of the line can be an enhancement for later.

**Comment by birtony** [comment]
Moved the focus to the end of the line as well. Let me know if you'd like it to be pulled?

**Pull request merged**